browser/js/components/Comment/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+import React, { Component, PropTypes } from 'react';
+import Paper from 'material-ui/Paper'
+import RaisedButton from 'material-ui/RaisedButton'
+import FlatButton from 'material-ui/FlatButton'
+import FontIcon from 'material-ui/FontIcon'
+import { Card, CardActions, CardHeader, CardText } from 'material-ui/Card'
+import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton'
+import { connect } from 'react-redux';
+import TextField from 'material-ui/TextField';
+import IconButton from 'material-ui/IconButton';
+import MoreVertIcon from 'material-ui/svg-icons/navigation/more-vert';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import {grey400, darkBlack, lightBlack} from 'material-ui/styles/colors';
+import {Toolbar, ToolbarGroup, ToolbarSeparator, ToolbarTitle} from 'material-ui/Toolbar';
+import {List, ListItem} from 'material-ui/List';
+import ModeEdit from 'material-ui/svg-icons/editor/mode-edit';
+
+const styles = {
```

Comment:
a note on this: a lot of React Components have a feature that receives styling based on passing in an object. so you'll have a class, but then you'll have something like `labelStyle={style.label}`. Also, we've refactored our styles as a separate style export "sheet".